ts/Core/Series/Point.ts

```diff
@@ -978,7 +978,9 @@ class Point {
         function update(): void {
 
             point.applyOptions(options);
-            point.resolveColor();
+            if (!series.options.colorByPoint) {
```

Comment:
It seems, checking the `series.options.colorByPoint` option is already done in `resolveColor()`. Perhaps we should fix something there?

Reply:
But I kind of need to skip it only on `update`, to fix a node update regression introduced by #15878. There might be some better way to deal with `colorCounter` though.

Also noticing `nodeLookup` is stored but doesnt actually get used the next time `generatePoints` runs, so all the nodes get re-created every time, not sure if intentional 🤔.

Reply:
> But I kind of need to skip it only on `update`, to fix a node update regression introduced by #15878. There might be some better way to deal with `colorCounter` though.

Ah, I see. Is the dependency wheel the only series with this regression? Perhaps #15878 should add `resolveColor()` in candlestick's `point.update`?

> Also noticing nodeLookup is stored but doesnt actually get used the next time generatePoints runs, so all the nodes get re-created every time, not sure if intentional 🤔.

No idea 😅 But `networkgraph` series uses this lookup object to add/remove nodes, so it's expected that `nodesLuookup` is up to date. Of course, instead of recreating the whole structure (and nodes), updating would be faster

Reply:
Anything `colorByPoint`: https://jsfiddle.net/u1mc8tdh/2/, `resolveColor` on `update` could probably still be useful in other cases.